Xamarin.Forms.Platform.Android/Renderers/DatePickerRenderer.cs

```diff
@@ -113,20 +113,17 @@ internal override void OnFocusChangeRequested(object sender, VisualElement.Focus
 			}
 		}
 
-		void CreateDatePickerDialog(int year, int month, int day)
+		protected virtual DatePickerDialog CreateDatePickerDialog(int year, int month, int day)
 		{
 			DatePicker view = Element;
-			_dialog = new DatePickerDialog(Context, (o, e) =>
+			var dialog = new DatePickerDialog(Context, (o, e) =>
 			{
 				view.Date = e.Date;
 				((IElementController)view).SetValueFromRenderer(VisualElement.IsFocusedPropertyKey, false);
 				Control.ClearFocus();
-
-				if (Forms.IsLollipopOrNewer)
```

Comment:
are we sure we don't need this now ?

Reply:
This event in unsubscribed on focus changed and when the renderer is disposed.